examples/README.md

```diff
@@ -341,6 +341,8 @@ This example creates 3 different docker images:
   certificate and sets a cron that renews the certificate (the cron
   runs every minute for testing purposes).
 
+On Linux, you will need the `libpcsclite-dev` package.
+
```

Comment:
@tashian: `libpcsclite-dev` is only required if we want to add YubiKey support, it's experimental, and I don't think you want to use it with docker, and it's very possible that you cannot use it.

With `CGO_ENABLED=0` they YubiKey support will be disabled and that package won't be necessary.

Reply:
👍 I removed it.